src/dtgtk/paint.c

```diff
@@ -270,19 +270,18 @@ void dtgtk_cairo_paint_switch(cairo_t *cr, gint x, gint y, gint w, gint h, gint
   const gint s = w < h ? w : h;
   cairo_translate(cr, x + (w / 2.0) - (s / 2.0), y + (h / 2.0) - (s / 2.0));
   cairo_scale(cr, s, s);
-
-  cairo_set_line_width(cr, 0.1);
   cairo_set_line_cap(cr, CAIRO_LINE_CAP_ROUND);
   if(flags & CPF_FOCUS)
   {
-    cairo_arc(cr, 0.5, 0.5, 0.66, 0, 2 * M_PI);
+    cairo_set_line_width(cr, 0.05);
+    cairo_arc(cr, 0.5, 0.5, 0.60, (-50 * 3.145 / 180), (230 * 3.145 / 180));
```

Comment:
I would even go with:
`cairo_arc(cr, 0.5, 0.5, 0.60, 0, (180 * 3.145 / 180));`

Reply:
I don't care that much. Personally i don't like half circles and here tried to keep the icon's shape.